### ISSUE TITLE
Missing "./isChromatic" specifier in "chromatic" package

### ISSUE THREAD

**Issue by chepetime** [chromaui/chromatic-cli#767]
Updated Chromatic to 6.19.5 and `isChromatic` is missing from the package.

```sh
ERR! Error: Missing "./isChromatic" specifier in "chromatic" package
```

<img width="793" alt="Captura de pantalla 2023-06-13 a la(s) 17 45 20" src="https://github.com/chromaui/chromatic-cli/assets/6405305/7235bfd0-aeee-4f72-93cc-d936d5f9697c">

Happens in ubuntu-latest with node LTS.

**Comment by elektronik2k5** [comment]
Can confirm, this happens to me too in all `6.19` versions, up to and including `6.19.5`. Reverting back to `6.18.2` fixes the issue.

**Comment by tmeasday** [comment]
Fixed in `chromatic@6.19.6`.

**Comment by chepetime** [comment]
Thanks @tmeasday! Can confirm it's fixed in `chromatic@6.19.6`.
